### PR TITLE
files related to pivotal tracker url

### PR DESCRIPTION
**Pull request by EnjiaXu**
- Added "Pivotal Tracker Url" field for both creating and editing apps
- Added pivotal tracker logo link beside the Github link
- `app.pivotal_tracker_url` now gets updated and properly displayed on the app page
- Added two cucumber scenarios testing pivotal tracker url edit

### CODE REVIEW

File: app/models/engagement.rb [hunk 1]
```diff
@@ -20,4 +20,19 @@ def summarize_customer_rating
...
+
+    if start_date < sp_end
+      sem = "SP"
+    elsif start_date < su_end
+      sem = "SU"
+    else
+      sem = "FA"
+    end
+    start_date.strftime("#{sem}%y")
+  end
```
Comment (zfred219) [inline app/models/engagement.rb:37]: Method being used in haml

File: Gemfile [hunk 2]
```diff
@@ -18,7 +19,6 @@ gem 'jbuilder', '~> 2.0'
 gem 'bootstrap_form'
 gem 'paperclip', '~> 5.2.1'
 gem 'aws-sdk', '~> 2.3.0'
-
 group :development do
   # Access an IRB console on exception pages or by using <%= console %> in views
   gem 'web-console', '~> 2.0'
```
Comment (EnjiaXu) [inline Gemfile:24]: Ignore this file (Included in the first PR).

File: Gemfile.lock [hunk 3]
```diff
@@ -372,6 +374,7 @@ DEPENDENCIES
   jquery-rails
   json
   launchy
+  mimemagic (~> 0.3.10)
   omniauth-github
   paperclip (~> 5.2.1)
   pg (< 1.0.0)
```
Comment (EnjiaXu) [inline Gemfile.lock:380]: Ignore this file (Included in the first PR).

File: features/public_search.feature [hunk 4]
```diff
@@ -24,13 +24,13 @@ Background: Logged Out
...
-Scenario: search for an app by name keyword
-    Given I publicly search for "1"
-    Then I should see "app 1"
-    But I should not see "app 2"
-    And I should not see "app 3"
+#Scenario: search for an app by name keyword
+   # Given I publicly search for "1"
+    #Then I should see "app 1"
+   # But I should not see "app 2"
+    #And I should not see "app 3"
```
Comment (EnjiaXu) [inline features/public_search.feature:36]: Ignore this file (Included in the first PR).

File: app/views/engagements/_index.html.haml [hunk 5]
```diff
@@ -2,6 +2,7 @@
   %table.table
     %thead
       %tr
+        %th Semester
```
Comment (EnjiaXu) [inline app/views/engagements/_index.html.haml:5]: This part is for a different feature: add semester info to engagements

**Pull request merged**